src/os/bluestore/BlueFS.cc

```diff
@@ -222,6 +242,13 @@ int BlueFS::reclaim_blocks(unsigned id, uint64_t want,
   return 0;
 }
 
+void BlueFS::handle_discard(unsigned dev, interval_set<uint64_t>& to_release)
+{
+  dout(10) << __func__ << dendl;
```

Comment:
May be output dev here as well?

Reply:
@ifed01 
I don't understand what you said.
Could you say more concretely?

Reply:
I just meant that you can additionally print value of 'dev' var here.

Reply:
I updated the log message.
Thanks for your comment!